sample_data/russia_by_federal_subject_since_2008/russia_by_federal_subject_since_2008.md

```diff
@@ -23,6 +23,30 @@ geojson_write(
   crs = NULL)
 ```
 
+Afterwards, use a text editor to change
+
+```
+[179.983337,68.96875],[180,68.949738],[180,68.855209],[180,67.499535],[180,67.498718],[180,65.984116],[179.99791,65.027084],
+```
+
+to
+
+```
+[179.983337,68.96875],[180,68.949738],[180,68.855209],[180,67.996353],[180,67.499535],[180,67.498718],[180,66.030731],[180,65.984116],[179.99791,65.027084],
+```
+
+and
+
+```
+[-179.998428,65.048439],[-180,66.030731],[-180,67.498718],[-180,67.996353],[-179.99791,68.979164],
+```
+
+to
+
+```
+[-179.998428,65.048439],[-180,66.030731],[-180,67.498718],[-180,67.499535],[-180,67.996353],[-179.99791,68.979164],
+```
+
```

Comment:
Should we state the reason why are we making this manual change to the coordinates?